### PR TITLE
New method ElementUtils.getSimpleNameOrDescription

### PR DESCRIPTION
**Pull request by mernst**
I'm not thrilled about the method name; suggestions for improving it are welcome.

### CODE REVIEW

File: javacutil/src/main/java/org/checkerframework/javacutil/ElementUtils.java [hunk 1]
```diff
@@ -205,6 +205,25 @@ public static String getSimpleName(ExecutableElement element) {
...
     }
 
+    /**
+     * Returns a user-friendly name for the given method. Does not return {@code "<init>"} or {@code
+     * "<clinit>"} as ExecutableElement.getSimpleName() does.
+     *
+     * @param element a method declaration
+     * @return a user-friendly name for the method
+     */
+    public static CharSequence getFriendlyName(ExecutableElement element) {
```
Comment (smillst) [inline javacutil/src/main/java/org/checkerframework/javacutil/ElementUtils.java:215]: I agree this name isn't great.  I think `getUserFriendlyName` would be an improvement.  Here are a few other, mostly bad names that might spark an idea of a better name:

* `getUserFriendlySimpleName`
* `getUserFriendlyName`
* `getPlainSimpleName`
* `getUserUnderstandableSimpleName`
* `getPrintableSimpleName`
* ` getLegibleSimpleName`

**Review by smillst — APPROVED** [review]
The code changes look good.  I added a comment about the name.

File: javacutil/src/main/java/org/checkerframework/javacutil/ElementUtils.java (hunk 1 above)
Comment (mernst) [reply javacutil/src/main/java/org/checkerframework/javacutil/ElementUtils.java:215]: I like `getUserFriendlyName` best among these suggestions (and it's better than the current name).
Another idea is:
 * `getSimpleNameOrDescription`

Comment (smillst) [reply javacutil/src/main/java/org/checkerframework/javacutil/ElementUtils.java:215]: I like `getSimpleNameOrDescription` best.

**Pull request merged**